modules/learn/pages/data/durability.adoc

```diff
@@ -81,6 +81,7 @@ Accordingly, it will be written to disk on those nodes.
 * _Timeout_.
 The number of milliseconds within which the durability requirements must be met.
 The SDK specifies a default of 2500.
+If the number is exceeded, the entire synchronous write is aborted.
```

Comment:
See comment on other commit - don't believe the synchronous write is aborted.  Might want to confirm with Graham.